model/labels/labels.go

```diff
@@ -350,6 +350,19 @@ func New(ls ...Label) Labels {
 	return set
 }
 
+// FromMaps returns new sorted Labels from the given maps, overriding each other in order.
+func FromMaps(maps ...map[string]string) Labels {
+	mLables := make(map[string]string)
```

Comment:
```suggestion
	mLabels := make(map[string]string)
```

Btw. we mostly only use `new` when there's a benefit to doing so over just the literal initialization, so I'd go for this here:

```suggestion
	mLabels := map[string]string{}
```